### ISSUE TITLE
onnx2ncnn TopK ScatterElements Greater Cast not supported yet

### ISSUE THREAD

**Issue by onexuan** [Tencent/ncnn#2485]
TopK not supported yet!
  # axis=1
  # largest=1
  # sorted=0
ScatterElements not supported yet!
  # axis=1
Greater not supported yet!
Cast not supported yet!
  # to=1

**Comment by nihui** [comment]
hi, please provide more info about your model, eg the project github, how is the model exported from pytorch

If your model is a tensorflow one, try keras2ncnn or mlir2ncnn. Never use onnx as the interop

**Comment by onexuan** [comment]
@nihui https://github.com/PeterL1n/BackgroundMattingV2 
onnx model https://drive.google.com/drive/folders/1cbetlrKREitIgjnIikG1HdM4x72FtgBh

**Comment by nihui** [comment]
针对onnx模型转换的各种问题，推荐使用最新的pnnx工具转换到ncnn
In view of various problems in onnx model conversion, it is recommended to use the latest pnnx tool to convert your model to ncnn

```shell
pip install pnnx
pnnx model.onnx inputshape=[1,3,224,224]
```

详细参考文档
Detailed reference documentation
https://github.com/pnnx/pnnx
https://github.com/Tencent/ncnn/wiki/use-ncnn-with-pytorch-or-onnx#how-to-use-pnnx
